libs/libcommon/src/libcommon/config.py

```diff
@@ -35,7 +36,9 @@
     PROCESSING_STEP_SPLIT_OPT_IN_OUT_URLS_COUNT_VERSION,
     PROCESSING_STEP_SPLIT_OPT_IN_OUT_URLS_SCAN_VERSION,
 )
-from libcommon.processing_graph import ProcessingGraphSpecification
+
+if TYPE_CHECKING:
+    from libcommon.processing_graph import ProcessingGraphSpecification
```

Comment:
why the change?

Reply:
To avoid circular dependencies

Reply:
hmmm, ok. I imagine that's why I used the "specification" field at that time. Maybe it's better to move `ProcessingStepSpecification` and `ProcessingGraphSpecification` to a types file, no?

Reply:
I think it's fine.

It's common for config files to import other types using TYPE_CHECKING to make sure every file can still import the config

Reply:
OKOK